pom.xml

```diff
@@ -366,6 +366,7 @@
 
                 <module>web-modules</module>
                 <module>persistence-modules/deltaspike</module> <!-- delta spike it doesn't yet the jakarta API-->
+                <module>persistence-modules/hibernate-ogm</module> <!-- hibernate-ogm wasn't updated and not support jakarta API -->
```

Comment:
we need to create a "lowest" priority jira to track this

Reply:
I created here a Jira task with lowest priority: https://team.baeldung.com/browse/JAVA-20557